.circleci/soltest.sh

```diff
@@ -75,6 +75,7 @@ do
         "--color_output=no"
         "--show_progress=yes"
         "--logger=JUNIT,error,test_results/$(get_logfile_basename "$((CPUs * CIRCLE_NODE_INDEX + run))").xml"
+        "--logger=HRF,error,stderr"
```

Comment:
Regarding https://github.com/ethereum/solidity/issues/13066#issuecomment-1138951346, I think these should go to stdout because they're part of the normal output of the command, not problems in `soltest` itself:
```suggestion
        "--logger=HRF,error,stdout"
```

But I'd also be fine with printing them to stderr if you can think of a good use case that would benefit from having them there rather on stdout.

Reply:
I set stderr for no special reason and can't think of anything to prevent it being changed to stdout... I am going to change it as you suggested.